LayoutTests/fast/repaint/justify-items-legacy-change-expected.txt

```diff
@@ -1,11 +1,11 @@
 Tests invalidation on justify-items style change (legacy value). Passes if green bars are centerd inside their red container.
 
 (repaint rects
-  (rect 1 53 50 50)
-  (rect 0 52 52 100)
-  (rect 151 53 50 50)
-  (rect 150 52 52 100)
-  (rect 0 52 251 300)
-  (rect 0 52 300 400)
+  (rect 51 53 50 50)
```

Comment:
The old expected values here were wrong because we didn't have `justify-self` implemented for block-level elements

Reply:
I would just update the test and remove the previously no-op justify-self

Reply:
Sorry, I don't think I understand: remove the justify-self: center from the test? But isn't that what the test is testing, the difference between `justify-items: center` and `justify-items: legacy center`?

Reply:
Ah I guess we don't use `justify-self` in that test. Nevermind me.